doc/user/content/sql/create-cluster.md

```diff
@@ -105,16 +105,18 @@ _replica_name_ | A name for a cluster replica.
 
 {{% replica-options %}}
 
-{{< note >}}
-If you do not specify an availability zone, Materialize will automatically
-assign the availability zone with the least existing replicas for the
-associated cluster to increase the cluster's tolerance to availability zone
-failure.
-
-To check the availability zone associated with each replica in a cluster, use
-the [`mz_cluster_replicas`](/sql/system-catalog/mz_catalog/#mz_cluster_replicas)
-system table.
-{{< /note >}}
+## Availability Zones and Scheduling
+
+When scheduling replicas, Materialize makes the following guarantees, within each cluster:
+
+- Different replicas are _never_ scheduled on the same node.
+- Different replicas are _always_ spread evenly across availability zones.
+  - Replicas with more than 1 process are excluded from this constraint. See [`mz_internal.mz_cluster_replica_sizes`](/sql/system-catalog/mz_internal/#mz_cluster_replica_sizes)
+  to determine if that is the case for your replicas.
+- Replicas with a more than 1 process _prefer_ being placed in the same availability zones.
+  - This is because multi-process replicas are shared-fate.
+- Replicas with a specific `AVAILABILITY ZONE` are _always_ scheduled in that availability zones.
+  - These replicas still participate in spread constraints.
```

Comment:
I think we should just axe this and restrict our explanation to managed cluster replicas. It's sooo hard to explain how the replicas are considered in other replica's spread constraints but also pin themselves to their own AZ...

```suggestion
```